app/controllers/games_controller.rb

```
@@ -52,4 +56,52 @@ def schedule
+      @games.reverse! if direction == 'desc'
+    when 'moneyline'
+      @games = games.to_a.sort_by { |g| g.current_bet_recommendations.find { |r| r.bet_type == 'moneyline' }&.ev || -999 }
+      @games.reverse! if direction == 'desc'
+    when 'total'
+      @games = games.to_a.sort_by { |g| g.current_bet_recommendations.find { |r| r.bet_type == 'total' }&.ev || -999 }
+      @games.reverse! if direction == 'desc'
+    else
+      @games = games.sort_by(&:start_time)
+    end
+    @games
+  end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for `sort_games` is too high. [<11, 26, 27> 39.06/22]

(Metrics/AbcSize)